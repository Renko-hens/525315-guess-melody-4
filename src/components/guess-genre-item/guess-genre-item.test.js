import React from "react";
import renderer from "react-test-renderer";
import GuessGenreScreenItem from "./guess-genre-item.jsx";

const answer = {
  src: `https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg`,
  genre: `rock`,
};

describe(`Render GuessGenreScreenItem components`, () => {
  it(`Render GuessGenreScreenItem`, () => {
    const tree = renderer.create((
      <GuessGenreScreenItem
        answer={answer}
        id={0}
        onChange={() => {}}
        renderPlayer={() => {}}
        userAnswer={false}
      />), {
      createNodeMock: () => {
        return {};
      }
    }).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
