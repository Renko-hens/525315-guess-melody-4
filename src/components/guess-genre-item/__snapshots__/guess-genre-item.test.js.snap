// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render GuessGenreScreenItem components Render GuessGenreScreenItem 1`] = `
<div
  className="track"
>
  <div
    className="game__answer"
  >
    <input
      checked={false}
      className="game__input visually-hidden"
      id="answer-0"
      name="answer"
      onChange={[Function]}
      type="checkbox"
    />
    <label
      className="game__check"
      htmlFor="answer-0"
    >
      Отметить
    </label>
  </div>
</div>
`;
